### PR TITLE
Pass module name and AWS region to integration tests

### PR DESCRIPTION
**Pull request by atsikham**
These changes allow to pass module name and AWS region to tests to identify which pipeline created resources are related to and in which region they are created. Task - #41

### CODE REVIEW

File: integration_test.go [hunk 1]
```diff
@@ -75,15 +76,15 @@ awsks:
       instance_type: t2.small
       asg_desired_capacity: 1
       asg_min_size: 1
-      asg_max_size: 1 
-`,
+      asg_max_size: 1
+`, defaultRegion),
```
Comment (mkyc) [inline integration_test.go:80]: is that correct? What if I run tests on different region?

**Review by mkyc — CHANGES_REQUESTED** [review]
One general question.

File: integration_test.go (hunk 1 above)
Comment (atsikham) [reply integration_test.go:80]: Yes, as this "init with defaults" test checks defaults and changing default value will require changing this value in tests.

Comment (mkyc) [reply integration_test.go:80]: Ah yeah, sorry I didn't notice that you use it in `init` part only.

**Review by mkyc — APPROVED** [review]
LGTM

**Review by seriva — APPROVED** [review]

**Pull request merged**